client/components/CustomBankField.js

```diff
@@ -0,0 +1,66 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
```

Comment:
Thank you for making a new component! I think its API would be clearer if it accepted two parameters: `name` and `value`, instead of a single `params` that has to take an unknown shape (you can deduce it below, but it is better to make it straightforward clear at the API level). Do you mind changing this, please?